dvc/repo/checkout.py

```diff
@@ -30,11 +31,14 @@ def checkout(self, target=None, with_deps=False, force=False, recursive=False):
     with self.state:
         _cleanup_unused_links(self, all_stages)
 
-        for stage in stages:
+        for index, stage in enumerate(stages):
             if stage.locked:
                 logger.warning(
                     "DVC file '{path}' is locked. Its dependencies are"
                     " not going to be checked out.".format(path=stage.relpath)
                 )
 
             stage.checkout(force=force)
+            progress.update_target(
```

Comment:
It seems more natural to use this progress bar for individual files and not stages, because a single stage might have 100K files in a directory, which will take a while and will defeat the purpose of this progress bar.